packages/backend/src/apps/lettersg/helpers/attachment.ts

```diff
@@ -0,0 +1,27 @@
+import { IGlobalVariable } from '@plumber/types'
+
+import { COMMON_S3_BUCKET, putObject } from '@/helpers/s3'
+
+export async function downloadAndStoreAttachmentInS3(
+  $: IGlobalVariable,
+  publicId: string,
+): Promise<string> {
+  // Note: no try catch because no known error found yet: will still throw general error
+  // separated the two http calls to avoid confusion
+  // Letters provide a redirect link so axios will auto-download it after redirecting
+  const { data: pdfData } = await $.http.get(`/v1/letters/${publicId}/pdfs`, {
+    responseType: 'arraybuffer',
+    headers: {
+      'Content-Type': 'application/json',
+      Accept: 'application/pdf',
+    },
+  })
```

Comment:
nit: if you merge latest develop into your branch, you can use url path params for this

```suggestion
  const { data: pdfData } = await $.http.get(`/v1/letters/:publicId/pdfs`, {
    responseType: 'arraybuffer',
    urlPathParams: {
      publicId
    },
    headers: {
      Accept: 'application/pdf',
    },
  })
```

Reply:
ohya okkkk thanks!